syntax = "proto3";
package feed.v1;
option go_package = "feedpb";

message FeedItem {
    string title = 1;
    string color = 2;
    string text = 3;
    string provider_name = 4;
    int32 score = 5;
}

message FeedGetRequest {
    int32 limit = 1;
}

message FeedGetResponse {
    repeated FeedItem items = 1;
}
